build/avalanchego-apis/admin-api-c-chain.md

```diff
@@ -0,0 +1,186 @@
+# Admin API (C-Chain)
```

Comment:
The C-Chain admin API should go on the existing C-Chain API page

Reply:
Okay 👍🏾